src/lib/annotations/Annotator.js

```diff
@@ -75,6 +77,13 @@ class Annotator extends EventEmitter {
             });
         }
 
+        // Destroy all annotate buttons
+        Object.keys(this.modeButtons).forEach((type) => {
```

Comment:
Are there cases in which modeButtons is null or undefined?
Perhaps this should be wrapped just to make sure its safe (just like line 72 above)
`if (this.modeButtons) {}`